drivers/gpio/Kconfig.lmp90xxx

```diff
@@ -0,0 +1,27 @@
+# LMP90xxx GPIO configuration options
+
+# Copyright (c) 2019 Vestas Wind Systems A/S
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig GPIO_LMP90XXX
+	bool "LMP90xxx GPIO driver"
+	depends on ADC_LMP90XXX_GPIO
+	help
+	  Enable GPIO driver for LMP90xxx.
+
+	  The LMP90xxx is a multi-channel, low power sensor analog
+	  frontend (AFE).
+
+	  The GPIO port of the LMP90xxx (D6 to D0) is exposed as a
+	  GPIO controller driver with read/write support.
+
+if GPIO_LMP90XXX
+
+config GPIO_LMP90XXX_INIT_PRIORITY
+	int "Driver init priority"
+	default 99
+	help
+	  Device driver initialization priority. This driver must be
+	  initialized after the LMP90xxx ADC driver.
```

Comment:
I tried that. I cannot get it to work. I keep getting `Undefined initialization levels used.` during linking.

Reply:
Right. I wasn't aware it was not possible to simply do `+<something>` on the priority since it was passed to `STRINGIFY` and made a part of the corresponding linker section name.
But perhaps you could move the option with the GPIO driver initialization priority to the ADC Kconfig, as it is anyway dependent on `ADC_LMP90XXX_GPIO`?

Reply:
I would rather keep it close to the GPIO driver where people are more likely to look. I have added a compile-time assert to the GPIO driver to ensure the initialisation order is correct.